src/main/java/com/stripe/net/ApiResource.java

```diff
@@ -85,17 +86,22 @@ public static String urlEncode(String str) {
     // Preserve original behavior that passing null for an object id will lead
     // to us actually making a request to /v1/foo/null
     if (str == null) {
+      BranchCoverageUtil.insertXAtIndex(4);
```

Comment:
Good use of that function! Exactly as expected :)

Reply:
![Thank you](https://media1.tenor.com/m/BMkCv-UCFIgAAAAC/bear-byebear.gif)